docs/src/pages/docs/crocks/Writer.md

```diff
@@ -0,0 +1,352 @@
+---
+title: "Writer"
+description: "Writer Crock"
+layout: "guide"
+functions: ["log", "read"]
+weight: 160
+---
+
+
+```haskell
+Monoid m => Writer m a
+```
+
+The `Writer` monad makes it possible to represent and compose computations that produce some 
+additional data along with their computed values. `Writer` is a `Product` type that consists 
+of a `Monoid` as the first value that typically holds metadata, while the 
+second value holds the result of a computation. 
+
+<article id="topic-implements">
+
+## Implements
+`Functor`, `Apply`, `Chain`, `Applicative`, `Monad`
+
+</article>
+
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Writer :: MonoidTypeRep M, Monoid m => M -> (m, a) -> Writer m a
```

Comment:
What is MonoidTypeRep? i've never seen this notation before

Reply:
Basically refers to the constructor of a monoid instance. You can find a similar use case in `ReaderT`.